data/themes/classic/style.css

```diff
@@ -115,8 +115,8 @@ PianoRoll {
 	qproperty-gridColor: rgb( 128, 128, 128 );
 	qproperty-noteModeColor: rgb( 255, 255, 255 );
 	qproperty-noteColor: rgb( 119, 199, 216 );
-	qproperty-noteBorderRadiusX: 5;
-	qproperty-noteBorderRadiusY: 2;
+	qproperty-noteOpacity: 128;
+	qproperty-borderlessNotes: false; /* boolean property, set true to have borderless notes */
```

Comment:
`borderlessNotes` is confusing as it causes a double negative. How about something like `useNoteBorders` with inverse logic?